lib/node_modules/@stdlib/time/base/parse-duration/docs/types/index.d.ts

```diff
@@ -0,0 +1,70 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2022 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 2.0
+
+/**
+* Interface describing a duration object.
+*/
+interface Duration {
+	/**
+	* Number of days.
+	*/
+	'days': number;
+
+	/**
+	* Number of hours.
+	*/
+	'hours': number;
+
+	/**
+	* Number of minutes.
+	*/
+	'minutes': number;
+
+	/**
+	* Number of seconds.
+	*/
+	'seconds': number;
+
+	/**
+	* Number of milliseconds.
+	*/
+	'milliseconds': number;
+}
+
+/**
+* Parses a duration string such as `1m3s10ms` into an object with `days`, `hours`, `minutes`, `seconds`, and `milliseconds`.
+*
```

Comment:
We should probably add a "Notes" section here and copy over the notes from the README, so that users can see them from the comforts of their IDE.

Reply:
Added to both TypeScript definition and implementation file.